patches/@projecttacoma+node-fhir-server-core+2.2.8.patch

```diff
@@ -1,16 +0,0 @@
-diff --git a/node_modules/@projecttacoma/node-fhir-server-core/dist/server/resources/4_0_0/schemas/datarequirement.js b/node_modules/@projecttacoma/node-fhir-server-core/dist/server/resources/4_0_0/schemas/datarequirement.js
```

Comment:
why was this patch deleted? Has the bug with datarequirment been fixed in the package?

Reply:
oh interesting, I didn't know this was useable code

Reply:
sorry about that, I misunderstood what the patch meant- just cherry-picked the commit and removed the removal of the patch and force pushed